projects/igniteui-angular/src/lib/core/styles/components/list/_list-theme.scss

```diff
@@ -36,7 +36,7 @@
         header-text-color: igx-color($palette, 'secondary'),
         item-background: #fff,
         item-text-color: igx-color($palette, 'grays', 800),
-        item-background-active: igx-color($palette, 'grays', 100),
+        item-background-active: #fff,
```

Comment:
Until now a moving/active list-item had a semi-transparent background (maybe added for some visual effect). After adding the left/right panning feature the active list-item should not be (semi) transparent because when the user starts dragging a list item, the whole panning template under it will be visible and this is not an effect we want to have. I added a hard-coded solid white background just like the list and the list-item are having.